e2etests/cypress/e2e/graphPiro.feature

```diff
@@ -2,4 +2,11 @@ Feature: Validation of Graph in PIRO
 
   Scenario: Validation of Graph in PIRO web on selecting input params
 
-    Given I open the piro website
\ No newline at end of file
+    Given I login the piro website
+
+    When I enter value of Target Compound mp-id
```

Comment:
can we check the flow for invalid compound values
rather than graph the error message should be visible

Reply:
Below ticket# that I created yesterday are for covering such validation. This particular story is to automate positive flow
JIRA# TRI-31159
JIRA# TRI-31160